.release-please-manifest.json

```
@@ -1,14 +1,15 @@
 {
-  ".": "1.3.0",
-  "packages/api-explorer": "0.9.3",
-  "packages/extension-api-explorer": "21.4.2",
+  ".": "1.4.0",
+  "packages/api-explorer": "0.9.4",
+  "packages/code-editor": "1.0.0",
+  "packages/extension-api-explorer": "21.4.3",
   "packages/extension-sdk": "21.4.2",
   "packages/extension-sdk-react": "21.4.2",
   "packages/hackathon": "21.4.2",
-  "packages/run-it": "0.9.3",
+  "packages/run-it": "0.9.4",
   "packages/sdk": "21.4.1",
-  "packages/sdk-codegen": "21.0.13",
-  "packages/sdk-codegen-scripts": "21.0.13",
+  "packages/sdk-codegen": "21.1.0",
```

Something got missed here.
```suggestion
  "packages/sdk-codegen": "21.1.14",
```